gltf-json/src/path.rs

```diff
@@ -10,8 +10,8 @@
 use std::fmt;
 
 /// An immutable JSON source path.
-#[derive(Clone, Debug)]
-pub struct Path(String);
+#[derive(Clone, Debug, PartialEq)]
+pub struct Path(pub String);
```

Comment:
I'd rather keep this private for now. Would `Deref<Target = str>` or a function `into_string()` be a reasonable replacement?